packages/core/src/components/Alert/Alert.scss

```diff
@@ -63,6 +68,10 @@ $alert-icon-size: $spacer-5;
   padding-left: $alert-icon-size + $alert-bar-size;
 }
 
+.ds-c-alert--hide-icon .ds-c-alert__body {
+  padding-left: $alert-padding;
```

Comment:
I think we could remove the padding from this element completely so that the spacing is consistent on all sides:

![image](https://user-images.githubusercontent.com/371943/32146136-6d1a8d28-bca9-11e7-86be-d87fbc800058.png)